bundles/core/src/main/java/com/adobe/cq/wcm/core/components/util/ComponentUtils.java

```diff
@@ -168,6 +194,9 @@ private static String generateId(@NotNull final Resource resource,
      */
     @NotNull
     public static String generateId(@NotNull final String prefix, @NotNull final String path) {
+        if ("search".equals(prefix)) {
```

Comment:
Is this logging needed?

Reply:
Good catch? It was a debugging left-over, removed it in 4d14728.